src/organizations/components/OrgProfileTab/LeaveOrg.tsx

```diff
@@ -1,36 +1,41 @@
 // Libraries
 import React, {FC, useContext} from 'react'
-import {useSelector} from 'react-redux'
+import {useDispatch, useSelector} from 'react-redux'
 
 // Components
-import {
-  ButtonShape,
-  ComponentColor,
-  ConfirmationButton,
-  FlexBox,
-  IconFont,
-} from '@influxdata/clockface'
+import {Button, ComponentColor, FlexBox, IconFont} from '@influxdata/clockface'
 
 // Selector
 import {selectCurrentOrg, selectUser} from 'src/identity/selectors'
 
 // Providers
 import {UsersContext} from 'src/users/context/users'
 
-// Constants
-import {CLOUD_URL} from 'src/shared/constants'
+// Overlay
+import {dismissOverlay, showOverlay} from 'src/overlays/actions/overlays'
+
+// Types
+import {RemoveMemberOverlayParams} from 'src/users/components/RemoveMemberOverlay'
 
 // Styles
 import 'src/organizations/components/OrgProfileTab/style.scss'
 
 export const LeaveOrgButton: FC = () => {
   const org = useSelector(selectCurrentOrg)
+  const {users} = useContext(UsersContext)
   const currentUserId = useSelector(selectUser)?.id
   const {removeUser} = useContext(UsersContext)
+  const userToRemove = users.find(user => user.id === currentUserId)
+  const dispatch = useDispatch()
 
   const handleRemoveUser = () => {
-    removeUser(currentUserId)
-    window.location.href = CLOUD_URL
```

Comment:
Don't need this `window.location.href`. The user will get redirected by the remove-user overlay if removing themselves from the org.